### ISSUE TITLE
[Med] Chrome: Screen reader: Action performed by the button is not announced by Screen reader.

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#9425]
___

## Action Performed:
1. Using Chrome+JAWS, open URL: staging.new.expensify.com
2. Login with the credentials.
3. On the homepage, select any chat.
4. Navigate to the visually looking PIN button and select it which is located at top right besides Call button.
5. Observe if the Screen reader is making any announcement.

## Expected Result:
After clicking on the visually looking Pin button or by selecting the Pin button,the Screen Reader should announce that the selected chat has been pinned or unpinned.

## Actual Result:
After Clicking or Selecting the visually looking Pin button, the Screen Reader is not announcing its status as it is pinned or unpinned.

## Workaround:
Yes

## Area issue was found in:
At chats section beside the call button 

## Failed WCAG checkpoints
4.1.2

## User impact:
The Screen reader users will not be able to know the status of the Pin button, whether the chat has been pinned or unpinned and thus they will not able to use this feature.

## Suggested resolution:
Ensure that the Screen reader announces the status of the chat(pinned/unpinned), when the user selects or clicks the Pin image button. Alternatively usage of aria-live attribute can announce the status as desired.

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:*v1.1.76-2*
**Reproducible in staging?:*Yes*
**Reproducible in production?:*Yes*
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:*Sumit*

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

https://user-images.githubusercontent.com/98817464/173504926-04e43956-598f-43f3-845f-8eb4cec7e4ad.mp4

**Comment by SumitDiyora** [comment]
https://user-images.githubusercontent.com/98817464/173504788-83630e4e-5910-4dcb-a134-2810c075b205.mp4
